### PR TITLE
Added Light Switch Status Bar Item

### PR DESCRIPTION
**Pull request by WretchedDade**
It was mentioned in #12 that it would be nice to have a GUI tool to switch themes. This PR implements a statusBarItem that when clicked calls the toggleThemes command switching the user's themes.

Here's a screenshot of the statusBarItem. I have it positioned on the far left. Let me know if it should a different icon/text or be positioned differently! The available icons can be found [here](https://code.visualstudio.com/api/references/icons-in-labels#grid-listing).

image

### CODE REVIEW

File: src/commands/binder.ts [hunk 1]
```diff
@@ -31,3 +38,22 @@ export function registerCommandSetThemeDay(
...
+
+/**
+ * Creates and returns the Light Switch Status Bar Item
+ *
+ * Note: The returned StatusBarItem still needs to be pushed into vscode's subscriptions
+ */
+export function createLightSwitchStatusBarItem(): StatusBarItem {
+  const statusBarItem = window.createStatusBarItem(
+    StatusBarAlignment.Left,
+    100 // The higher the number the farther left it is on the status bar
```
Comment (timrodz) [inline src/commands/binder.ts]: 1. Could you set the type for `statusBarItem`? It should be `StatusBarItem`.
2. Could you `100` into a const variable called `priority`?

File: src/commands/binder.ts [hunk 2]
```diff
@@ -31,3 +38,22 @@ export function registerCommandSetThemeDay(
...
+  );
+
+  statusBarItem.command = `lightSwitch.toggleThemess`;
```
Comment (timrodz) [inline src/commands/binder.ts]: 1. There's a typo here - it should be `toggleThemes` (Minus one `s`).
2. Can you load `lightSwitch` from `util/Constants`?

**Review by timrodz — CHANGES_REQUESTED** [review]
Hi @WretchedDade, thanks so much for your awesome contribution! I left you some feedback which would be awesome if you could action it — Great job! 💯

**Comment by WretchedDade** [comment]
@timrodz, good catches! I have made the requested changes.

**Review by timrodz — APPROVED** [review]
Awesome work, @WretchedDade — Merging! 🔥

**Pull request merged**